egs/heroico/s5/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
+                           Version 2.0, January 2004
```

Comment:
I don't like including the Apache license here--  It makes it look almost like it's a separate project, which will lead to confusion.
There is already a COPYING file at the top of the Kaldi project which includes the full text of the Apache 2.0 license (after some preamble).